content/en/docs/components/training/reference/architecture.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "Architecture"
+description = "The Training Operator Architecture"
+weight = 10
++++
+
+{{% stable-status %}}
+
+## What is the Training Operator Architecture?
+
+The original design was drafted in April 2021 and is [available here for reference](https://docs.google.com/document/d/1x1JPDQfDMIbnoQRftDH1IzGU0qvHGSU4W6Jl4rJLPhI/).
+The goal was to provide a unified Kubernetes operator that supports multiple
+machine learning/deep learning frameworks. This was done by having a "Frontend"
+operator that decomposed the job into different configurable Kubernetes
```

Comment:
As you can see, we decided to move forward with Option 1, which is monolithic operator to orchestrate all CRDs for different frameworks. E.g. we don't use the frontend operator. Maybe we should explain it here.

https://docs.google.com/document/d/1x1JPDQfDMIbnoQRftDH1IzGU0qvHGSU4W6Jl4rJLPhI/edit#heading=h.qhgaj6b27svz